### DISCUSSION TITLE
Does the  Image support nv12 ?

### DISCUSSION THREAD

**Discussion by jerry-xie-000** [foxglove/community#888] [Q&A]
I only found the Image format for yuv are:   uyvy or yuv422 、  yuyv or yuv422_yuy2.

However, many camera data is formatted as nv12. If change nv12 to rgb,  will double the data size. It's not suitable for embedded devices.

**Comment by defunctzombie** [comment]
We do not have support for nv12. The supported formats are listed here: https://docs.foxglove.dev/docs/visualization/panels/image#supported-encodings

**Reply by torsten-schuenemann-keen** [reply]
@defunctzombie
Will there be support for NV12 in the near future?
Since Mai 2025 it is standard in ROS2: https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/include/sensor_msgs/image_encodings.hpp

**Comment by defunctzombie** [comment]
Tracking issue: https://linear.app/foxglove/issue/ERT-1929/add-nv12-image-encoding-support-to-image-panel

**Comment by defunctzombie** [comment]
We've added NV12 support. It is now available in the web app and will be available in the next desktop release (2.49.0)
